crates/bitwarden-exporters/src/cxf/ssh.rs

```diff
@@ -96,4 +135,39 @@ mod tests {
         assert_eq!(fields[2].value.as_deref(), Some("2025-01-01"));
         assert_eq!(fields[3].value.as_deref(), Some("Generated using OpenSSH"));
     }
+
+    #[test]
+    fn test_try_into_credentials() {
+        let ssh = SshKey {
+            private_key: "-----BEGIN OPENSSH PRIVATE KEY-----\nb3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAAAMwAAAAtzc2gtZW\nQyNTUxOQAAACAyQo22TXXNqvF+L8jUSSNeu8UqrsDjvf9pwIwDC9ML6gAAAJDSHpL60h6S\n+gAAAAtzc2gtZWQyNTUxOQAAACAyQo22TXXNqvF+L8jUSSNeu8UqrsDjvf9pwIwDC9ML6g\nAAAECLdlFLIJbEiFo/f0ROdXMNZAPHGPNhvbbftaPsUZEjaDJCjbZNdc2q8X4vyNRJI167\nxSquwOO9/2nAjAML0wvqAAAAB3Rlc3RrZXkBAgMEBQY=\n-----END OPENSSH PRIVATE KEY-----\n".to_string(),
```

Comment:
I'm surprised Github didn't make a stink about this. I feel like some bot always false positives when I commit test keys to vc.